Gopkg.toml

```diff
@@ -233,3 +233,11 @@
 [[constraint]]
   branch = "master"
   name = "golang.org/x/oauth2"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/docker/libnetwork"
+
+[[constraint]]
```

Comment:
I'm no dep expert, so be skeptical of this comment, but should we change this to an override to indicate that its not a direct dependency?

Reply:
Yes, as per my understanding of https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md , it makes sense to capture this as an `override`

Reply:
```suggestion
[[override]]
```